coq.opam.template

```diff
@@ -1,23 +1 @@
-build: [
-  ["dune" "subst"] {dev}
-  [ "./configure"
-    "-release" # -release must be the first command line argument
-    "-prefix" prefix
-    "-mandir" man
-    "-libdir" "%{lib}%/coq"
-    "-native-compiler" "yes" {rocq-native:installed} "no" {!rocq-native:installed}
-  ] {with-test}
-  [
-    "dune"
-    "build"
-    "-p"
-    name
-    "-j"
-    jobs
-    "--promote-install-files=false"
-    "@install"
-    "@runtest" {with-test}
-    "@doc" {with-doc}
-  ]
-  ["dune" "install" "-p" name "--create-install-files" name]
-]
+build: [ ]
```

Comment:
I'm not sure about this, the tests may have problems on opam CI but at least for 4.14 that was just missing conf- packages (conf-time and conf-python3) AFAICT

Reply:
(NB the tests are still in package coq https://github.com/rocq-prover/rocq/blob/24d5c80cf4164934e2d12d16b06c86014e944172/dune#L87 and btw depend on coqide-server)

Reply:
Well, this coq package is just a metapackage, it shouldn't build nor install anything. If there are tests to run, they should probably be run somewhere else?

Reply:
this package got removed from this repo so this doesn't matter anymore